brjs-core-tests/src/test/java/org/bladerunnerjs/plugin/commands/core/HelpCommandTest.java

```diff
@@ -19,8 +19,8 @@
 	@Before
 	public void initTestObjects() throws Exception
 	{
-		command1 = new MockCommandPlugin("command1", "Command #1 description.", "command-usage", "Command #1 help.");
-		command2 = new MockCommandPlugin("command2", "Command #2 description.", "command-usage", "Command #2 help.");
+		command1 = new MockCommandPlugin("command1", "Command #1 description.", "command-usage", "  Command #1 help.");
+		command2 = new MockCommandPlugin("command2", "Command #2 description.", "command-usage", "  Command #2 help.");
```

Comment:
As far as I can tell these lines are configuring mock commands that are added to the model so they can be used in tests. The third argument sets the command's help message text, instead of having a mock command object with a fixed help message. So this test is now not simulating the behaviour of a real commands help message format since the help message itself has extra whitespace rather than a utility that outputs them to the console adding the extra whitespace.